### ISSUE TITLE
Reproducing GCPR results

### ISSUE THREAD

**Issue by andy8744** [hukkelas/DeepPrivacy#35]
Thanks for the detailed write up of your amazing work. I have a few questions, in particular relating to  "Image Inpainting with Learnable Feature Imputation". 

1. In [GCPR.md](https://github.com/hukkelas/DeepPrivacy/blob/master/GCPR.md) mask_infer.py is referenced but I can't seem to find it in this github repo. 
2. I'm trying to reproduce the results in the paper "Image Inpainting with Learnable Feature Imputation", would it be possible to train a new model from scratch, providing I have the training and mask dataset.

**Comment by andy8744** [comment]
Thanks for the update. It seems like I get an error from running mask_infer.py
I get the error: `mask_infer.py: error: unrecognized arguments: -c`

I looked at the code and I assume this is because `deep_privacy/config/Config.py` and `deep_privacy/config/default_parser.py` does not exist.

**Comment by hukkelas** [comment]
They are both defined in [`__init__.py`](https://github.com/hukkelas/DeepPrivacy/blob/master/deep_privacy/config/__init__.py).

I think the GCPR readme is slightly incorrect; no need for the -c argument. Try the following:

```
python3 mask_infer.py config_path.py -i image_file.png -m mask_path -t target_path
```

**Comment by andy8744** [comment]
Thanks! 
I got the error ```[WARNI] Could not find checkpoint. outputs\gcpr\places\places_B```
But after modifying `_get_checkpoint_path()` to explicitly return the ckpt file, I got it to work. 
https://github.com/hukkelas/DeepPrivacy/blob/de30f547428277385db6b0536c9b74750f56e1eb/deep_privacy/engine/checkpointer.py#L35-L42

One other question I have is, is it possible to train with your own dataset and masks. For example if have my own images and mask for each specific image, where would I have to modify the code to train a new inpainting model. Thanks again for your time.
